@uportal/content-carousel/src/components/ContentCarousel.html

```diff
@@ -24,6 +24,7 @@
                   v-if="item.imageUrl"
                   :src="item.imageUrl"
                   :alt="item.altText"
+                  v-bind:style="{ height }"
```

Comment:
```suggestion
                  :style="{ height }"
```

https://vuejs.org/v2/guide/syntax.html#v-bind-Shorthand